web/src/components/ak-page-navbar.ts

```diff
@@ -303,32 +283,40 @@ export class AKPageNavbar
 
     #toggleSidebar() {
         this.open = !this.open;
-
-        this.dispatchEvent(
-            new CustomEvent<SidebarToggleEventDetail>("sidebar-toggle", {
-                bubbles: true,
-                composed: true,
-                detail: { open: this.open },
-            }),
-        );
+        this.dispatchEvent(new PageNavMenuToggle(!!this.open));
     }
 
     //#endregion
 
+    //#region Event Handlers
+
+    #onWebSocket = () => {
+        this.firstUpdated();
+    };
+
+    #onPageDetails = (ev: PageDetailsUpdate) => {
```

Comment:
In the parlance of, *input change event*, *page details change event*
```suggestion
    #pageDetailsListener = (event: PageDetailsChangeEvent) => {
```